docs/source/science/NISAR/Simulated_NISAR.ipynb

```diff
@@ -0,0 +1,11823 @@
+{
```

Comment:
<p>Typo: <code>sfs.S3FileSystem</code> should be <code>s3fs.S3FileSystem</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>